libs/dyn/dynvar/ref_test.go

```diff
@@ -36,8 +40,6 @@ func TestNewRefInvalidPattern(t *testing.T) {
 		"${_-_._-_.id}",                 // cannot use _- in sequence
 		"${0helloworld.world-world}",    // interpolated first section shouldn't start with number
 		"${helloworld.9world-world}",    // interpolated second section shouldn't start with number
-		"${a-a.a-_a-a.id}",              // fails because of -_ in the second segment
```

Comment:
is this pattern "-_" supported? I don't see it in tests.

Reply:
@denik yes, it is supported and the test case is here https://github.com/databricks/cli/pull/2203/files#diff-17c69a46b43fd7c1f660dd95fb007b3e00e33068c5b666226aa545f592ce34ebR24